src/client/components/EditUserForm/CollaboratorPermissions/CollaboratorPermissions.tsx

```diff
@@ -0,0 +1,81 @@
+import './CollaboratorPermissions.scss'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Objects } from '@utils/objects'
+
+import { SubSection } from '@meta/assessment'
+import { CollaboratorProps, User } from '@meta/user'
+
+import { useAssessmentSections } from '@client/store/assessment'
+import CollaboratorAccessModal from '@client/components/CollaboratorAccessModal'
+
+type Props = {
+  user: User
+}
+
+const CollaboratorPermissions = (props: Props) => {
+  const { user } = props
+  const { i18n } = useTranslation()
+
+  const assessmentSections = useAssessmentSections()
+
+  const options = assessmentSections
+    .reduce((prev, curr) => [...prev, ...curr.subSections], [])
+    .filter((subSection: SubSection) => subSection.props.anchor)
+    .reduce((prev, curr) => ({ ...prev, [curr.uuid]: curr.props.anchor }), {})
+
+  const properties = (user.roles[0].props as CollaboratorProps) || undefined
+  const sections = Objects.isEmpty(properties) ? 'none' : properties.sections
+
+  const tableDataPermissions = Object.entries(sections)
+    .filter(([_, v]) => v.tableData === true)
+    .map(([k, _]) => options[k])
+    .sort()
+    .join(', ')
+
+  const descriptionsPermissions = Object.entries(sections)
+    .filter(([_, v]) => v.descriptions === true)
+    .map(([k, _]) => options[k])
+    .sort()
+    .join(', ')
+
+  const [modalOptions, setModalOptions] = useState<{ open: boolean }>({ open: false })
+
+  const _onEditPermissionsClick = () => {
+    setModalOptions({ open: true })
+  }
+
+  const _onEditPermissionsClose = () => {
+    setModalOptions({ open: false })
+  }
+
+  return (
+    <div className="edit-user__form-item edit-user__form-item-permissions">
+      <div className="edit-user__form-label">{i18n.t<string>('Permissions')}</div>
```

Comment:
what's this??

Reply:
<img width="531" alt="Schermata 2022-09-30 alle 12 44 32" src="https://user-images.githubusercontent.com/10047945/193253562-efdb6cbf-c314-4388-aaa7-b1df9a8551c4.png">

Reply:
I meant using 'Permissions' is not consistent on how we write label key in the js translation files. fix it

Reply:
fixed